website/pages/docs/job-specification/hcl2/functions/conversion/index.mdx

```diff
@@ -0,0 +1,6 @@
+---
+layout: docs
+page_title: conversion - Functions - Configuration Language
+sidebar_title: Type Conversion Functions
+description: Overview of available conversion functions
```

Comment:
These empty index pages result in empty `Overview` pages in the sidebar. Can we eliminate them without breaking navigation or are we stuck with them b/c of the framework?

Reply:
Apparently, I could have just added the name fields to the sidebar nav: done in https://github.com/hashicorp/nomad/pull/9322/commits/f8b4cfc2410395fc7058302c4c958c319d55b513